### ISSUE TITLE
Searching for variants in Alt contigs

### ISSUE THREAD

**Issue by rhdolin** [elimuinformatics/vcf2fhir#12]
Need to decide if this is something we should support. If, for instance, you are looking for HLA variants, you might need to search across all HLA Alt contigs. If, for instance, you used an alternate-locus aware variant caller, you may not find variants in the alt contig regions using this algorithm, as they may have been aligned instead with the Alt contigs.

Alt contigs are generally created for regions known to be highly polymorphic (e.g. HLA) and/or known to vary considerably across populations. There may be many query liftover challenges (e.g. many Alt contigs are new in b38). We need to understand how Alt contigs version.

**Comment by Rohan-cod** [comment]
I would like to work on this issue. Can I take this up?

**Comment by rhdolin** [comment]
@Rohan-cod Absolutely. Give me a day or two to put together some reading material that will describe this issue in more detail.
